docs/stacks-guide-aws.md

```diff
@@ -0,0 +1,911 @@
+---
+title: "Crossplane Stacks Guide: AWS Setup" toc: true weight: 332 indent: true
+---
+
+# Crossplane Stacks Guide: AWS Setup
+
+## Table of Contents
+
+  - [Crossplane Stacks Guide: AWS Setup](#crossplane-stacks-guide-aws-setup)
+  - [Table of Contents](#table-of-contents)
+  - [Introduction](#introduction)
+  - [Before you begin](#before-you-begin)
+  - [Install AWS stack](#install-aws-stack)
+  - [Configure AWS account](#configure-aws-account)
+  - [Configure Crossplane Provider for
+    AWS](#configure-crossplane-provider-for-aws)
+  - [Set Up Network Configuration](#set-up-network-configuration)
+  - [Configure Provider Resources](#configure-provider-resources)
+  - [Recap](#recap)
+
+## Introduction
+In this guide we are focusing on WordPress workload, and will show how it could
+be deployed using AWS resources. WordPress workload uses a generic Kubernetes
+cluster and a MySQL database as its resource, which are not tied to a specific
+provider (also known as **resource claims**). Instead, these resource claims
+could be configured to use a be satisfied by a specific cloud provider.
+
+We will walk you through installing the AWS stack, configuring an aws account,
+and setting up cloud-specific resources in order to create and configure a
+network configuration in which these cloud resources can communicate with each
+other. Once the network configuration is done, we will specify a resource class
+for each resource claim type. Then we will create WordPress workload, which will
+create related external resources for each claim in AWS.
+
+
+## Before you begin
+Before continuing in this guide, make sure that all the following are true:
+- Crossplane is installed 
+- [crossplane-cli] is installed
+- `kubectl` is configured to target the Crossplane cluster
+- An aws account is available
+
+
+Then make a place to work:
+```bash
+WORK_DIR=$(mktemp -d)
+```
+
+## Install AWS stack
+
+Once Crossplane is installed, it can be extended to support AWS by installing
+its *stack*. 
+
+The namespace where we install the stack, is also the one that our managed AWS
+resources will reside. Let's call this namespace `infra-aws`, and go ahead and
+create it:
+
+```bash
+# the namespace that the aws infra structure resources will be created
+INFRA_NAMESPACE=infra-aws
+# create the namespace in Crossplane
+kubectl create namespace ${INFRA_NAMESPACE}
+```
+<details> <summary>output</summary>
```

Comment:
This collapsible `details` functionality is pretty cool! Do you know if it works well when compiled for the website?

Reply:
I don't. It's part of standard github markdown though:
https://gist.github.com/ericclemmons/b146fe5da72ca1f706b2ef72a20ac39d

Reply:
I just tested this, and it does not work as expected on the fully rendered website in all cases, so let's remove it for now and revisit it after the release. The nested ones later on in the document seem to get all weird, so everything in the document from "VPC" onward is a collapsed block with deeper and deeper nesting.

Reply:
Removed the details tag. However this seems to be very useful for keeping our documents manageable.`Kubebuilder` is also using this style in their doc site. We should later look into enabling this in the site.

Reply:
I'm not able to merge this PR until @hasheddan's requested changes have been addressed. I think he might need to explicitly approve, but I'm going to try marking this thread resolved to see if that does the trick.